fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/services/CreateResourceServiceImplTest.java

```diff
@@ -166,6 +170,7 @@ public void setUp() {
         nonRdfSourceOperationFactory = new NonRdfSourceOperationFactoryImpl();
         setField(createResourceService, "nonRdfSourceOperationFactory", nonRdfSourceOperationFactory);
         setField(createResourceService, "containmentIndex", containmentIndex);
+        createResourceService.eventAccumulator = eventAccumulator;
```

Comment:
Just curious why you departed from using setField(... ?

Reply:
For the sake of consistency, I would suggest we do all the assignments of protected members in the same way - at least within a test class.

Reply:
Changed